src/Forms/UsuarioType.php

```diff
@@ -38,7 +38,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 ]
             ])
             ->add('sexo', TextType::class, [
-                'label' => 'Sexto',
+                'label' => 'Sexo',
```

Comment:
Ok Rafael isso conta pra vc como pontos sim!!! todo ajuste correto qu se faz nos pull request eu to consideranco.